### PR TITLE
Add basic go script

### PR DESCRIPTION
**Pull request by mr-stringer**

### CODE REVIEW

**Review by mr-stringer — COMMENTED** [review]
@mpagot I made some changes to your markdown that were causing some linting errors.  Can you let me know if you are happy for me to merge them?

**Comment by mpagot** [comment]
> @mpagot I made some changes to your markdown that were causing some linting errors. Can you let me know if you are happy for me to merge them?

I'm ok with these changes

File: .gitignore [hunk 1]
```diff
@@ -16,4 +16,6 @@ ansible/playbooks/roles/sap_storage
 /terraform/libvirt/.terraform.lock.hcl
 /terraform/gcp/.terraform.tfstate.lock.info
 ansible/playbooks/go.sh
+ansible/playbooks/destroy.sh
```
Comment (mpagot) [inline .gitignore:19]: Are these two line to be removed later?

File: README.md [hunk 2]
```diff
@@ -5,40 +5,74 @@ This project aims to provide automation that will create highly-available SAP de
...
+
+To get started you must should create a new `variables.sh`:
+
+```shell
+cp variables.example variables.sh
+```
+
+Edit the values of variables.sh to match your configuration.
+
+Then copy the `azure_hana_media.example.yaml` file and edit the values so the ansible knows where to download the installation media.  For azure, it is preferred upload the media to blobs in an Azure storage account
```
Comment (mpagot) [inline README.md:66]: capital Ansible and Azure

**Pull request merged**